packages/story-editor/src/app/story/effects/useHistoryEntry.js

```diff
@@ -30,6 +30,7 @@ import deleteNestedKeys from '../utils/deleteNestedKeys';
 // if only one (or multiple) of these properties change but nothing else changes.
 // These are still saved as part of changes involving other properties.
 const ELEMENT_PROPS_TO_IGNORE = [
+  'resource.font',
```

Comment:
Why would a resource have a property of font?

Reply:
I'll remove this --- it was listed in the original issue.